src/OpenTelemetry.Exporter.Geneva/GenevaExporterHelperExtensions.cs

```diff
@@ -16,20 +16,45 @@
 
 using System;
 using System.Diagnostics;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
 using OpenTelemetry.Internal;
 using OpenTelemetry.Trace;
 
 namespace OpenTelemetry.Exporter.Geneva;
 
 public static class GenevaExporterHelperExtensions
 {
+    /// <summary>
+    /// Adds <see cref="GenevaTraceExporter"/> to the <see cref="TracerProviderBuilder"/>.
+    /// </summary>
+    /// <param name="builder"><see cref="TracerProviderBuilder"/> builder to use.</param>
+    /// <param name="configure">Exporter configuration options.</param>
+    /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
     public static TracerProviderBuilder AddGenevaTraceExporter(this TracerProviderBuilder builder, Action<GenevaExporterOptions> configure)
+        => AddGenevaTraceExporter(builder, name: null, configure);
+
+    /// <summary>
+    /// Adds <see cref="GenevaTraceExporter"/> to the <see cref="TracerProviderBuilder"/>.
+    /// </summary>
+    /// <param name="builder"><see cref="TracerProviderBuilder"/> builder to use.</param>
+    /// /// <param name="name">Name which is used when retrieving options.</param>
+    /// <param name="configure">Exporter configuration options.</param>
+    /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+    public static TracerProviderBuilder AddGenevaTraceExporter(this TracerProviderBuilder builder, string name, Action<GenevaExporterOptions> configure)
     {
         Guard.ThrowIfNull(builder);
 
+        name ??= Options.DefaultName;
+
+        if (configure != null)
+        {
+            builder.ConfigureServices(services => services.Configure(name, configure));
```

Comment:
FYI this will work fine today for metrics & traces because they have different options classes. Logging appears to be sharing `GenevaExporterOptions` with tracing so if/when logging gets hooked up to DI/options there will be issues with the delegates potentially clobbering each other. Cross that bridge when we come to it 😄